vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/ExperimentDetailsComponent.java

```diff
@@ -127,214 +149,168 @@ private void initTagAndNotesLayout() {
     getContent().addContent(tagAndNotesLayout);
   }
 
-  private void initTabSheet(ExperimentInformationService experimentInformationService) {
-    initSummaryCardBoard(experimentInformationService);
-    initExperimentalGroupsBoard();
+  private void layoutTabSheet() {
     experimentSheet.add("Summary", experimentSummary);
     experimentSheet.add("Experimental Groups", contentExperimentalGroupsTab);
     getContent().addContent(experimentSheet);
     experimentSheet.setSizeFull();
   }
 
-  private void addCreationCard() {
-    experimentalGroupsCollection.addComponentAsFirst(experimentalGroupCreationCard);
-  }
-
-  private AddExperimentalGroupsDialog createExperimentalGroupDialog() {
-    AddExperimentalGroupsDialog dialog = new AddExperimentalGroupsDialog();
-    dialog.addExperimentalGroupSubmitListener(
-        groupSubmitted -> handler.onGroupSubmitted(groupSubmitted));
-    return dialog;
-  }
-
-  private void setUpCreationCard() {
+  private void configureExperimentalGroupCreation() {
     experimentalGroupCreationCard.addListener(this::handleEvent);
   }
 
-  private void handleEvent(DisclaimerConfirmedEvent disclaimerConfirmedEvent) {
-    experimentSheet.setSelectedIndex(0);
+  private void addCloseListenerForAddVariableDialog() {
+    addVariablesDialog.addOpenedChangeListener(it -> {
+      if (!it.isOpened()) {
+        experimentalVariableCard.refresh();
+      }
+    });
   }
 
-  private void initSummaryCardBoard(ExperimentInformationService experimentInformationService) {
-    initBlockingVariableCard();
-    initExperimentalVariableCard(experimentInformationService);
-    Row topRow = new Row(sampleOriginCard, blockingVariableCard);
-    Row bottomRow = new Row(experimentalVariableCard);
-    summaryCardBoard.add(topRow, bottomRow);
-    summaryCardBoard.setSizeFull();
+  private void handleGroupSubmittedSuccess() {
+    reloadExperimentalGroups();
+    experimentalGroupsDialog.close();
+  }
```

Comment:
NIT: Down the line the reload could be triggered by a dedicated subscription to an ExperimentalGroupRegistered event.